utils/lib/helpers.ts

```diff
@@ -27,9 +28,10 @@ export const prop =
  * @returns quickstarts that use the passed in component
  */
 export const getRelatedQuickstarts = (
-  localComponentPath: string
+  localComponentPath: string,
+  context?: QuickstartContext
 ): Quickstart[] =>
-  Quickstart.getAll().filter((q) =>
+  Quickstart.getAll(undefined, context ?? {}).filter((q) =>
```

Comment:
Can we/should we default the context on the static method to {} also instead of making it the callers responsibility?

Reply:
That seems like a much better approach.